apps/services/user-notification/src/app/modules/notifications/notificationsWorker/mocks.ts

```diff
@@ -135,20 +136,23 @@ const delegations: Record<string, DelegationRecordDTO[]> = {
       fromNationalId: userWithDelegations.nationalId,
       toNationalId: userWithNoDelegations.nationalId,
       subjectId: null, // test that 3rd party login is not used if subjectId is null
+      type: null,
```

Comment:
Is this the reason for typing it `| null` on the DTO? we should just set it here as `AuthDelegationType.ProcurationHolder` for these delegation test cases in the notifications worker.

Reply:
Fixed.